src/test/java/uk/gov/companieshouse/api/accounts/controller/CompanyAccountControllerTest.java

```diff
@@ -39,13 +43,22 @@
 
     @BeforeEach
     public void setUp(){
-        when(companyAccountService.save(companyAccount)).thenReturn(createdCompanyAccount);
+        try {
+            when(companyAccountService.save(any(CompanyAccount.class), anyString())).thenReturn(createdCompanyAccount);
```

Comment:
This should fail the tests if the mocking fails. It will be harder to debug if the error thrown in the test is caused by an exception which is logged separately.